### PR TITLE
LPS-69372 | master

### PR DESCRIPTION
**Pull request by vicnate5**
@leoadb @thiago-rocha @ctampoya

### CODE REVIEW

**Comment by brianchandotcom** [comment]
ci:reopen

**Comment by brianchandotcom** [comment]
@vicnate5 why did

Leave form?

get changed to

Leave form ?

Can you fix and resend? Thx.

File: modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -38,12 +39,9 @@ is-equal-to=Is equal to
...
 javax.portlet.title.com_liferay_dynamic_data_lists_form_web_portlet_DDLFormAdminPortlet=Forms
 javax.portlet.title.com_liferay_dynamic_data_lists_form_web_portlet_DDLFormPortlet=Form
-jump-to-page=Jump To Page
-show-x=Show {0}
-require-x=Require {0}
-enable-x=Enable {0}
 jump-from-x-to-x=Jump from page {0} to {1}
-leave-form=Leave form?
+jump-to-page=Jump To Page
+leave-form=Leave form ?
```
Comment (vicnate5) [inline modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/content/Language.properties]: @leoadb Was this change intended?

Comment (leoadb) [inline modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/content/Language.properties]: @vicnate5 There is no technical reason or even a requirement for that. I changed it because it looks better for me. I'm going to change it back and send to Brian. Thanks and sorry for that.

Comment (vicnate5) [inline modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/content/Language.properties]: Thanks @leoadb!
